### DISCUSSION TITLE
step by step execution of tpot in azure or ec2 instance

### DISCUSSION THREAD

**Discussion by Shashank292002** [telekom-security/tpotce#1554] [General]
can anyone provide step by step execution of deployment of tpot in the azure or ec2 instance cloud, and view in the web ui

**Comment by Ayaan7345** [comment]
The step-by-step process will depend on your understanding of cloud services. While it is not particularly difficult, I am confident you can manage it. Should you require assistance with manual setup, I would be glad to help. Alternatively, if you prefer to deploy the VM with all the networking configurations pre-established, you can find the Terraform code on my GitHub repository [here](https://github.com/Ayaan7345/Azure_SEIM_Lab). Im currently working on getting the tpot script to run with terraform as of right now you have to run the script by yourself.

Edit: 
I have updated the project its now fully automated. [here](https://github.com/Ayaan7345/Azure_SEIM_Lab)

**Reply by Ayaan7345** [reply]
Just sent the friend request.
